api/src/gcp.rs

```diff
@@ -573,12 +573,18 @@ impl FakeGcsTester {
 
     assert!(self.proc.is_none());
 
-    #[cfg(target_os = "macos")]
+    #[cfg(all(target_os = "macos", target_arch = "aarch64"))]
     let p = concat!(
       env!("CARGO_MANIFEST_DIR"),
       "/../tools/bin/darwin-arm64/fake-gcs-server"
     );
 
+    #[cfg(all(target_os = "macos", target_arch = "x86_64"))]
```

Comment:
after already introducing the binaries in https://github.com/jsr-io/jsr/pull/812, i've introduced this to enable tests on my machine.